root/static/scripts/relationship-editor/components/RelationshipDialogContent.js

```diff
@@ -526,23 +526,29 @@ const AttributesSection = (React.memo<AttributesSectionPropsT>(({
   const booleanRangeSelectionHandler =
     useRangeSelectionHandler('boolean');
 
-  return (attributesState.attributesList.length || canEditDates) ? (
+  const hasAttributes = attributesState.attributesList.length > 0;
+
+  return (hasAttributes || canEditDates) ? (
     <>
       <h2>
         <div className="heading-line" />
         <span className="heading-text">
           {l('Attributes')}
-          {' '}
-          {bracketed(
-            <a href="#" onClick={handleAttributesHelpClick}>
-              {l('help')}
-            </a>,
-          )}
+          {hasAttributes ? (
```

Comment:
This should probably be reverted if we're adding back the dates help (which we should).